### DISCUSSION TITLE
Set optimization flag with helm chart

### DISCUSSION THREAD

**Discussion by stigtermichiel** [kubevela/kubevela#3448] [Q&A]
As referenced in this issue: https://github.com/oam-dev/kubevela/issues/3296, it was recommended to set `--optimize-disable-component-revision`. 

We deploy vela-core with helm and saw that this flag is not exposed in the chart. Is there another way of doing it?

**Comment by wonderflow** [comment]
Yes, we should expose that in chart value, thanks for reporting.

**Reply by wonderflow** [reply]
I've created an issue for that #3459 

Would you like to contribute for the fix?

**Reply by stigtermichiel** [reply]
yes! Will do today/tomorrow
